pkg/aws/idps.go

```diff
@@ -71,3 +79,45 @@ func (c *awsClient) HasOpenIDConnectProvider(issuerURL string, accountID string)
 	}
 	return true, nil
 }
+
+func (c *awsClient) ListOpenIDConnectProviders(clusterID string, clusters []*cmv1.Cluster) ([]OIDC, error) {
+	// get all open id connect providers
+	oidcProviderOutput, err := c.iamClient.ListOpenIDConnectProviders(&iam.ListOpenIDConnectProvidersInput{})
+	if err != nil {
+		return nil, err
+	}
+
+	var oidcProviders []OIDC
+	for _, oidcp := range oidcProviderOutput.OpenIDConnectProviderList {
+		// filter out rosa oidc providers
+		if !strings.Contains(aws.StringValue(oidcp.Arn), "rh-oidc") {
+			continue
+		}
+
+		// split arn to get cluster id
+		splitOIDC := strings.Split(aws.StringValue(oidcp.Arn), "amazonaws.com/")
```

Comment:
@vkareh there are no tags added to oidc providers, will I add tags in this MR, to save splitting the arn to get the clusterID?